traffic_monitor_golang/traffic_monitor/threadsafe/polledcaches.go

```diff
@@ -103,11 +116,22 @@ func copyCaches(a map[enum.CacheName]struct{}) map[enum.CacheName]struct{} {
 	return b
 }
 
+func copyCachesTime(a map[enum.CacheName]time.Time) map[enum.CacheName]time.Time {
+	b := map[enum.CacheName]time.Time{}
+	for k, v := range a {
+		b[k] = v
+	}
+	return b
+}
+
+const PolledBytesPerSecTimeout = time.Second * 60
```

Comment:
"Trouble communicating with a cache" should, by itself, mark the cache as unhealthy.